pkgs/development/python-modules/mwxml/default.nix

```diff
@@ -12,23 +11,14 @@
 
 buildPythonPackage rec {
   pname = "mwxml";
-  version = "0.3.4";
+  version = "0.3.5";
   pyproject = true;
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-ejf3RfdwcEp0Ge+96dORuHS5Bx28GSs7H4HD1LUnde4=";
+    hash = "sha256-K/5c6BfX2Jo/jcKhCa3hCQ8PtWzqSFZ8xFqe1R/CSEs=";
   };
 
-  patches = [
-    # https://github.com/mediawiki-utilities/python-mwxml/pull/21
-    (fetchpatch2 {
-      name = "nose-to-pytest.patch";
-      url = "https://github.com/mediawiki-utilities/python-mwxml/compare/2b477be6aa9794064d03b5be38c7759d1570488b...71bbfd2b309e0720a34a4e783b71169aebc571ef.patch";
```

Comment:
(note for next time: when using `fetchpatch2` with Github urls, you must add `?full_index=1` to get a stable hash)

Reply:
Or rather, not use fetchpatch2 at all with these `compare`-based patches. Since they do not support full_index=1.